README.md

```diff
@@ -18,16 +18,28 @@ under the MIT license (see the accompanying `LICENSE.txt` file) and I
 have tried to make sure that this repository contains all of the scripts
 needed to download and set up the data set that we used.
 
-## Quick Start
+## Quick Start (🐳 Docker)
+```bash
+# clone and go inside the repo
```

Comment:
Could you pivot away from these comments, and match the sections above and below this one by using paragraphs and full sentences here in this section?

Reply:
Thanks for the link Brandon. It helped me a lot :)